Build/build.ps1

```diff
@@ -8,15 +8,18 @@ $all_ok = $True
 
 function Build-One {
     param(
-        $project
+        $project,
+        [switch]
+        $generateQir
     );
 
     Write-Host "##[info]Building $project..."
     dotnet build $project `
         -c $Env:BUILD_CONFIGURATION `
         -v $Env:BUILD_VERBOSITY `
         /property:DefineConstants=$Env:ASSEMBLY_CONSTANTS `
-        /property:Version=$Env:ASSEMBLY_VERSION
+        /property:Version=$Env:ASSEMBLY_VERSION `
+        /property:QirGeneration=($generateQir ? "true" : "false")
```

Comment:
It looks like the subshell substitution here requires a `$`, sorry for the typo.
```suggestion
        /property:QirGeneration=$($generateQir ? "true" : "false")
```

Reply:
Ah that did the trick. Thanks!